server/routes/swipeRoutes.js

```diff
@@ -92,68 +92,73 @@ router.post("/nextuser/:userid", async (req, res) => {
     console.log(err);
   }
 });
-router.post("/recommendation/:userid/:count?", async (req, res) => {
+router.post("/recommendation", async (req, res) => {
   try {
-    const userid = req.params.userid;
-    const count = Number(req.params.count || 20);
+    const userid = req.body.userid;
+    const count = Number(req.body.count || 20);
 
     if (typeof count === "number") {
       const { listOfUsers, write, data } = await getRecommendations({
         userid,
         count,
       });
 
-      res.status(200).send({
+      res.status(200).json({
         message: `User ${userid} queue addition successful: added ${data.recommendqueue.length} recommendations`,
         data: write,
         added: listOfUsers,
       });
     } else {
-      res.status(400).send({ message: "Bad request" });
+      res.status(400).json({ message: "Bad request" });
     }
   } catch (err) {
     console.log(err);
   }
 });
 
-router.post("/:userid/:otherid/:bool", async (req, res) => {
+router.post("/swipe", async (req, res) => {
   try {
-    const bool = req.params.bool;
-    const userid = req.params.userid;
-    const otherid = req.params.otherid;
+    const userid = req.body.userid;
+    const otherid = req.body.otherid;
+    let bool = req.body.bool;
 
-    console.log("swipe", userid, userid, bool, typeof bool);
-    if (bool === "true" || bool === "false") {
-      const data = await readFromDb({
-        key: "userid",
-        value: userid,
-        collection: Users,
-      });
-      console.log("userswipes", data.userswipes);
-      if (bool === "true") {
-        data.userswipes[otherid] = {
-          swipe: "true",
-          time: new Date().toISOString(),
-        };
-      } else if (bool === "false") {
-        data.userswipes[otherid] = {
-          swipe: "false",
-          time: new Date().toISOString(),
-        };
-      }
-      const write = await writeToDb({
-        userid,
-        userswipes: data.userswipes,
-        collection: Users,
-      });
+    if (bool === "true" || bool === true) {
+      bool = true;
+    } else if (bool === "false" || bool === false) {
+      bool = false;
+    } else {
+      res.status(400).json({ message: "Bad request" });
+    }
 
-      res.status(200).send({
-        message: `Swipe by ${userid} successful: ${bool} on ${otherid}`,
-        data: write,
-      });
+    // console.log("swipe", userid, userid, bool, typeof bool);
+
+    const data = await readFromDb({
+      key: "userid",
+      value: userid,
+      collection: Users,
+    });
+    // console.log("userswipes", data.userswipes);
+    if (bool) {
+      data.userswipes[otherid] = {
+        swipe: true,
+        time: new Date().toISOString(),
+      };
     } else {
-      res.status(400).send({ message: "Bad request" });
+      data.userswipes[otherid] = {
+        swipe: false,
+        time: new Date().toISOString(),
+      };
```

Comment:
Since both paths of the `if/else` logic are the same aside from the `swipe`, let's just use the `bool` in the swipe itself.

```suggestion
    data.userswipes[otherid] = {
      swipe: bool,
      time: new Date().toISOString(),
    };
```